libnmstate/nm/vxlan.py

```diff
@@ -59,11 +59,14 @@ def get_info(device):
         base_iface = ''
         if device.props.parent:
             base_iface = device.props.parent.get_iface()
+        remote = device.props.group
+        if not remote:
+            remote = ''
         return {
             VXLAN.CONFIG_SUBTREE: {
                 VXLAN.ID: device.props.id,
                 VXLAN.BASE_IFACE: base_iface,
-                VXLAN.REMOTE: device.props.group,
```

Comment:
don't want to be picky, but why not `VXLAN.REMOTE: device.props.group or '',` . That would allow you to remove all the above.

I think it's more concise. Anyway, not important.

Reply:
My fault @maiqueb , won't block it now that I messed up.

Reply:
I won't argue with someone that can use my own weight against me :laughing: